test/test_runner/test_parser.cpp

```diff
@@ -204,6 +219,15 @@ std::string TestParser::parseCommand() {
     return params.substr(1, params.size() - 2);
 }
 
+std::string TestParser::parseCommandRepeat() {
```

Comment:
My understanding on `repeat` is that it should be limited to strings, so can you add checks here on the param that it is string literal (i.e., wrapped with `""`)?

Reply:
added validation